oop/src/views/MoveInfoPage.vue

```diff
@@ -58,8 +70,42 @@ export default {
     DateInput,
     Input,
   },
+  data: () => {
+    return {
+      moveFromBCDate: null,
+      arriveDestinationDate: null,
+      showServerValidationError: false,
+      showDateValidationError: false,
+    }
+  },
+  created() {
+    this.moveFromBCDate = this.$store.state.form.moveFromBCDate;
+    this.arriveDestinationDate = this.$store.state.form.arriveDestinationDate;
+  },
+  validations() {
+    // Formatting validation that the arrival date must be after the move date
+    if (this.moveFromBCDate > this.arriveDestinationDate) {
+      this.showDateValidationError = true;
+    }
+    else {
+      this.showDateValidationError = false;
+    }
```

Comment:
The validations shouldn't be done this way. Have a look at how I did validations in the "oop-backup" project.

Validation usage: https://github.com/bcgov/MOH-AOP-OOP/blob/main/oop-backup/src/modules/enrolment/components/EnrolmentPersonalInfo.vue#L125-L130

Validators: https://github.com/bcgov/MOH-AOP-OOP/blob/main/oop/src/components/DateInput.vue#L56-L78

You'll need to add the `date-fns` package to the OOP "package.json" in order for the validators to work.